### PR TITLE
add fresh translations for locale/it_IT/electrum.po

### PR DESCRIPTION
**Pull request by zebra-lucky**
- add fresh translations for locale/it_IT/electrum.po

### CODE REVIEW

File: locale/it_IT/electrum.po [hunk 1]
```diff
@@ -1302,12 +1291,11 @@ msgstr "Confermato. Firma della transazione..."
 
 #: electrum_dash/gui/qt/protx_wizards.py:91
 msgid "Connect to registered DIP3 Masternode"
-msgstr ""
+msgstr "Collegarsi al  Masternode DIP3 registrato"
```
Comment (jardev) [inline locale/it_IT/electrum.po]: extra space

File: locale/it_IT/electrum.po [hunk 2]
```diff
@@ -38,17 +38,17 @@ msgstr " {} {} pronto per essere inviato."
...
 msgid "%s %s registered DIP3 Masternodes."
-msgstr ""
+msgstr "%s %s Masternode DIP3 registrati."
 
 #: electrum_dash/gui/qt/protx_qt.py:967
-#, fuzzy, python-format
+#, python-format
 msgid "%s Dip3 Masternode Info"
-msgstr "Masternode.conf"
+msgstr "%s Info Msternode Dip3"
```
Comment (jardev) [inline locale/it_IT/electrum.po]: a typo, must be `Masternode` instead of `Msternode`

File: locale/it_IT/electrum.po [hunk 3]
```diff
@@ -2065,7 +2048,7 @@ msgstr "Impostazioni e-mail"
 
 #: electrum_dash/gui/qt/dash_net_dialog.py:296
 msgid "Enable Dash Network"
-msgstr ""
+msgstr "Abilita Network Dash "
```
Comment (jardev) [inline locale/it_IT/electrum.po]: extra space at the end?

File: locale/it_IT/electrum.po [hunk 4]
```diff
@@ -4377,33 +4338,32 @@ msgstr "PIN impostato"
...
-#, fuzzy
 msgid "PS Type"
-msgstr "Tipo di seed"
+msgstr "Tipo PS"
 
 #: electrum_dash/gui/qt/address_list.py:378
-#, fuzzy
 msgid "PS Type:"
-msgstr "Tipo di seed"
+msgstr "Tipo PS"
```
Comment (jardev) [inline locale/it_IT/electrum.po]: no colon at the end

File: locale/it_IT/electrum.po [hunk 5]
```diff
@@ -6457,9 +6414,8 @@ msgid "This type of script is not supported with {}."
 msgstr "Questo tipo di script non è supportato con {}."
 
 #: electrum_dash/plugins/revealer/qt.py:265
-#, fuzzy
 msgid "This version supports a maximum of {} characters."
-msgstr "Questa versione supporta al massimo 216 caratteri."
+msgstr "Questa versione supporta un massimo di {} caratteri"
```
Comment (jardev) [inline locale/it_IT/electrum.po]: no period at the end

**Review by jardev — COMMENTED** [review]
Please fix a few small issues

File: locale/it_IT/electrum.po (hunk 1 above)
Comment (zebra-lucky) [reply locale/it_IT/electrum.po]: fixed in b028cf7

File: locale/it_IT/electrum.po (hunk 3 above)
Comment (zebra-lucky) [reply locale/it_IT/electrum.po]: fixed in b028cf7

File: locale/it_IT/electrum.po (hunk 4 above)
Comment (zebra-lucky) [reply locale/it_IT/electrum.po]: fixed in b028cf7

File: locale/it_IT/electrum.po (hunk 5 above)
Comment (zebra-lucky) [reply locale/it_IT/electrum.po]: fixed in b028cf7

File: locale/it_IT/electrum.po (hunk 2 above)
Comment (zebra-lucky) [reply locale/it_IT/electrum.po]: fixed in b028cf7

**Pull request merged**